ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -5047,7 +5047,7 @@ private static void deleteAndRename(FileSystem destFs, Path destFile, FileStatus
     try {
       // rename cannot overwrite non empty destination directory, so deleting the destination before renaming.
       destFs.delete(destFile);
-      LOG.info("Deleted destination file" + destFile.toUri());
+      LOG.info("Deleted destination file: " + destFile.toUri());
```

Comment:
Since you are touching it, can you change it to
LOG.info("Deleted destination file: {}", destFile.toUri());

Reply:
ack